nginx-log-metrics/README.md

```diff
@@ -0,0 +1,66 @@
+# Nginx log metrics demo
+
+## Quick Start
+
+Build and start with proxy:
+
+```shell
+docker compose build --build-arg https_proxy=http://172.17.0.1:7890
+docker compose up -d --force-recreate
+```
+
+## Structure
+
+<!-- This chart is generated by https://derlin.github.io/docker-compose-viz-mermaid/ -->
+
+```mermaid
```

Comment:
Maybe use draw.io to generate an SVG picture. The render version is hard to interpret.

Reply:
We can remove some detail like `config_data` to make the main components more clear